cmplibrary/src/preprod/java/com/sourcepoint/cmplibrary/creation/ComponentFactory.kt

```diff
@@ -1,7 +1,22 @@
 package com.sourcepoint.cmplibrary.creation
 
 import android.content.Context
+import com.sourcepoint.cmplibrary.data.network.NetworkClient
 import com.sourcepoint.cmplibrary.data.network.connection.ConnectionManager
 import com.sourcepoint.cmplibrary.data.network.connection.create
+import com.sourcepoint.cmplibrary.data.network.createNetworkClient
+import com.sourcepoint.cmplibrary.data.network.util.HttpUrlManagerSingleton
+import com.sourcepoint.cmplibrary.data.network.util.ResponseManager
+import com.sourcepoint.cmplibrary.exception.Logger
+import okhttp3.OkHttpClient
 
 internal fun getConnectionManager(appCtx: Context): ConnectionManager = ConnectionManager.create(appCtx)
+
+internal fun networkClient(appCtx: Context, netClient: OkHttpClient, responseManage: ResponseManager, logger: Logger): NetworkClient {
```

Comment:
why do we need and instance of `Context` here, since we don't use it?

Reply:
because using the variants technique, we must have the signature of the method the same in all files (the contents can be different)

Reply:
gotcha